Demo/Common/Networking/PaymentsRequest.swift

```diff
@@ -94,6 +95,7 @@ internal struct PaymentsRequest: APIRequest {
         case lineItems
         case delegatedAuthenticationData
         case recurringProcessingModel
+        case mandate
```

Comment:
What is this for? Seems to be a bigger than just PayTo change

Reply:
See the comment on the actual object. It is required for payto on API calls (both are on the demo code)